iOS Documentation Revamp (#650)
ParsePlatform/Docs: _includes/ios/files.md

```
</div>

##Deleting Files

> You can delete files using the [REST API]({{site.baseUrl}}/rest/guide/#deleting-files) using the files name. You will need to provide the master key in order to be allowed to delete a file.

Reviewer: ```suggestion
If you know the name of a file you can delete it using the [REST API]({{site.baseUrl}}/rest/guide/#deleting-files). Your master key is required for this operation.
```
